### DISCUSSION TITLE
Is it normal that this script only wipes the messages sent to others by me?

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#239] [1. Questions & Answers] [answered]
As said in the title, it does not delete messages I received from others, it only deletes the ones I sent to them.
When I change the Author ID to theirs and run the script, I get

> Error deleting message, API responded with status 403!

Is this how this script is intended to function? I want to fully delete ALL direct messages..

**Comment by Toby222** [comment, marked as answer]
[Status 403 means you're lacking permission.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403)
This script only automates deleting, it doesn't give you any permissions on Discord you don't have already.
You don't have permission to delete the other person's DMs.
You'd have to ask them to delete their half of the conversation.
